commit 00b438869cbe414d9a92419dcf702a0a74b679c5
Author: Brian Norris <briannorris@chromium.org>
Date:   Mon Jun 11 14:09:43 2018 -0700

    ath10k: snoc: use correct bus-specific pointer in RX retry
    
    [ Upstream commit 426a0f0b5a2fe1df3496ba299ee3521159dba302 ]
    
    We're 'ath10k_snoc', not 'ath10k_pci'. This probably means we're
    accessing junk data in ath10k_snoc_rx_replenish_retry(), unless
    'ath10k_snoc' and 'ath10k_pci' happen to have very similar struct
    layouts.
    
    Noticed by inspection.
    
    Fixes: d915105231ca ("ath10k: add hif rx methods for wcn3990")
    Signed-off-by: Brian Norris <briannorris@chromium.org>
    Signed-off-by: Kalle Valo <kvalo@codeaurora.org>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/net/wireless/ath/ath10k/snoc.c b/drivers/net/wireless/ath/ath10k/snoc.c
index a3a7042fe13a..aa621bf50a91 100644
--- a/drivers/net/wireless/ath/ath10k/snoc.c
+++ b/drivers/net/wireless/ath/ath10k/snoc.c
@@ -449,7 +449,7 @@ static void ath10k_snoc_htt_rx_cb(struct ath10k_ce_pipe *ce_state)
 
 static void ath10k_snoc_rx_replenish_retry(struct timer_list *t)
 {
-	struct ath10k_pci *ar_snoc = from_timer(ar_snoc, t, rx_post_retry);
+	struct ath10k_snoc *ar_snoc = from_timer(ar_snoc, t, rx_post_retry);
 	struct ath10k *ar = ar_snoc->ar;
 
 	ath10k_snoc_rx_post(ar);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1210_00b438869cbe414d9a92419dcf702a0a74b679c5.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1210_00b438869cbe414d9a92419dcf702a0a74b679c5.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
